src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Cvt.kt

```diff
@@ -75,37 +79,47 @@ class Cvt : AbstractCommand(
         val inputSplit = inputPattern.find(input) ?: return
```

Comment:
You've already checked if the input pattern matches - use `!!` instead. Or if you want, move the if statement block above to this line.

Reply:
What does `!!` do?

Reply:
> The third option is for NPE-lovers: the not-null assertion operator (`!!`) converts any value to a non-null type and throws an exception if the value is null. We can write `b!!`, and this will return a non-null value of `b` (e.g., a `String` in our example) or throw an NPE if `b` is null

https://kotlinlang.org/docs/reference/null-safety.html#the--operator

I think my latter suggestion in the original comment is more appropriate anyway.

Reply:
Ok, just wanted to know where you'd want me to place it

Reply:
I'm not going to do that since this returns instead of throwing an exception when an edge case happens

Reply:
I meant something like this, since you're essentially doing the check twice needlessly:
```kotlin
        // This
        val inputSplit = inputPattern.find(input) ?: {
            channel.sendMessage("<:NatsumiThink:701311512714805279> Not sure what you mean by `$input`.").queue()
            return
        }

        // Instead of 
        if (!input.matches(inputPattern)) {
            channel.sendMessage("<:NatsumiThink:701311512714805279> Not sure what you mean by `$input`.").queue()
            return
        }
        val inputSplit = inputPattern.find(input) ?: return
```
It's only a minor note...